### PR TITLE
Pr2965v2

### PR DESCRIPTION
**Pull request by DickBaker**
Roji, Hi

Hope this PR2965v2 will be acceptable (I removed editorconfig and Console.Read contentious bits).
NB I explicitly added SqlClient v2.1.1 (18/12/2020) as I recall you recommended it during an EF StandUp (despite not direct dependent in parent EFCoreSqlServer v5.0.1 package). I got some different BM Ratio etc that may relate so I added that to the PR commit [suggest you drop it before merge into master branch].

In response to your encouragement [thanks!] and as noted in earlier comment, I am working up a more complex perf scenario in a separate file some doesn't muddy the stream [or frighten the newts/newbies]. Am looking forward to your/AV/team response as results show some wild Ratios [yup, real-world experience in consulting-land], so I hope you/RoW will enjoy and not take fright and bury it [the censorship we are currently experiencing in UK]. Happy 2021 to the team!

### CODE REVIEW

File: samples/core/Benchmarks/AverageBlogRanking_20210107.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// * Summary *
```
Comment (smitpatel) [inline samples/core/Benchmarks/AverageBlogRanking_20210107.txt:1]: If this file is not used in doc pages then should remove it. It represent one snapshot of benchmark run. User may get confused seeing different data when they run.

Comment (roji) [reply samples/core/Benchmarks/AverageBlogRanking_20210107.txt:1]: Yeah, also note that there are also some results [inline on the page](https://docs.microsoft.com/en-us/ef/core/performance/performance-diagnosis?tabs=simple-logging%2Cload-entities#benchmarking-with-ef-core).

**Review by roji — COMMENTED** [review]
LGTM, will merge once AverageBlogRanking_20210107.txt is removed.

**Comment by DickBaker** [comment]
glad we are getting closer, and happy for that .txt file to be removed in the final cut [was just there to contrast the Ratio/etc results as on docco compared with my 1-off experience, and was just there to invite you to re-run "officially" and alter any docco as appropriate. BTW I forgot to flag the Issue #2964 (to be closed when you merge the revised Dickbits).

Anyway I am about to upload my next PR to answer my Issue #2991 to keep you guys amused. [please don't ask me to rebase it if/when you merge in the earlier bits]. Enjoy

File: samples/core/Benchmarks/Benchmarks.csproj [hunk 2]
```diff
@@ -1,12 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
-  </PropertyGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
```
Comment (smitpatel) [inline samples/core/Benchmarks/Benchmarks.csproj:2]: 2 space indentation in csproj files please.

Comment (DickBaker) [reply samples/core/Benchmarks/Benchmarks.csproj:2]: I suppose that an authoritative .editorconfig would have enforced this to give consistent result that we can all celebrate [hooray!]. Anyway, as PR Contributor I always tick the "Collaborators can red-ink and re-hack-at-will" [ok it only nearly says that] so go for pasteurised merge and then we can all move on

Comment (roji) [reply samples/core/Benchmarks/Benchmarks.csproj]: Am also removing the package reference to SqlClient. While it's indeed a good idea to use the latest SqlClient, I'd really rather keep our samples as simple as possible etc.

**Review by roji — APPROVED** [review]
Approving after fixup

**Comment by roji** [comment]
Thanks @DickBaker

**Pull request merged**